api/leaderboard/celery.py

```
@@ -422,3 +422,31 @@ def atcoder_user_update(self):
+            UnifiedScoreHistory.objects.update_or_create(
+                username=username,
+                date=today,
+                defaults={
+                    "total_score": float(row.get("total_score", 0)),
+                    "github_score": float(row.get("github_score", 0)),
+                    "cf_score": float(row.get("cf_score", 0)),
+                    "cc_score": float(row.get("cc_score", 0)),
+                    "lt_score": float(row.get("lt_score", 0)),
+                },
+            )
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)